CONTRIBUTING.rst

```diff
@@ -18,12 +18,71 @@ first in an appropriate venue, drafting a PEP and gathering feedback, and
 developing at least a prototype reference implementation of your idea.
 
 
+Contributing changes to existing PEPs
+-------------------------------------
+
+In general, PEPs that are not marked as Draft or Active are usually considered
+to be historical documents rather than documentation or living specifications.
+To propose a substantially revised version of the original ideas,
+you should generally consider opening a new PEP instead.
+As such, make sure to consider the type and status of the PEP(s)
+in question before proposing changes:
+
+* **Rejected**, **Deferred**, **Withdrawn** and **Superceded** PEPs should
```

Comment:
Not sure about Deferred or Withdrawn. Deferred PEPs could see updates to bring them up to date; and a Withdrawn PEP could be prepared for a new shot.

Reply:
I was initially going to say something like this here (see above), but my general impression (which might be off base) was that for old PEPs that got enough out of step with modern Python to earn such a status (as the great majority of Deferred and Withdrawn PEPs are, since only PEPs >=several years old seem to have either status, aside from PEPs withdrawn for other reasons) it was usually recommended to start fresh with a new PEP rather than completely re-writing an old one. But that might not be an accurate impression, since I can't point to somewhere in PEP 1 where it says that (as opposed to for Rejected PEPs).

In any case, I've revised this bullet point to offer revival as a possibility.